websites/W/Warframe Market/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.15",
+  "apiVersion": 1,
+  "author": {
+    "id": "1112774630416076850",
+    "name": "auxiliaryfrfr"
+  },
+  "service": "Warframe Market",
+  "description": {
+    "en": "Warframe Market is the leading Warframe trading platform for tenno, where you can buy and sell vaulted & unvaulted items along with Warframe sets, Parts, Weapons, Mods, and more!"
+  },
+  "url": "warframe.market",
+  "version": "1.0.0",
```

Comment:
Please add a `regExp` field. This will replace the functionality of the `url` field in the future.